src/qibolab/instruments/dummy.py

```diff
@@ -53,13 +53,26 @@ def get_values(self, options, sequence, exp_points):
                 values = np.random.rand(samples * exp_points) * 100 + 1j * np.random.rand(samples * exp_points) * 100
             elif options.acquisition_type is AcquisitionType.INTEGRATION:
                 values = np.random.rand(exp_points) * 100 + 1j * np.random.rand(exp_points) * 100
-            results[ro_pulse.qubit] = results[ro_pulse.serial] = options.results_type(values)
-        return results
+        return values
 
     def play(self, qubits: Dict[Union[str, int], Qubit], sequence: PulseSequence, options: ExecutionParameters):
```

Comment:
If we decide to keep everything in `play`, the following needs to be updated too:
```suggestion
    def play(self, qubits: Dict[Union[str, int], Qubit], sequence: Union[PulseSequence, List[PulseSequence]], options: ExecutionParameters):
```